packages/main/src/ColorPalette.ts

```diff
@@ -153,6 +153,13 @@ class ColorPalette extends UI5Element {
 	@property({ type: Boolean })
 	popupMode!: boolean;
 
+	/**
+	 * Defines if the palette is rendered on phone.
+	 * @private
+	 */
+	@property({ type: Boolean })
+	onPhone!: boolean;
```

Comment:
rename that one on phone or the other one placed in ColorPaletteItem for consistency

Reply:
I was planning on doing it in separate change so I don't mix it up with this one, but i can do it in this one as well.